src/app/pages/interest/interest.js

```diff
@@ -68,83 +75,51 @@ function BookSelectorPage({selectedBooksRef, bookBeingSubmitted}) {
             <FormInput
                 longLabel="How many students do you teach each semester?"
                 inputProps={{
-                    name: 'Number_of_Students__c',
+                    name: 'number_of_students',
                     type: 'number',
```

Comment:
just curious, but did SF get it's fields renamed so they don't use the double underscores now?